OONIProbeUnitTests/ResultTest.m

```diff
@@ -32,17 +32,18 @@ - (void)testNoMeasurements {
 }
 
 - (void)testNotUploaded {
-    //One measurement, isEveryMeasurementUploaded false
+    //One measurement, with is_done false, isEveryMeasurementUploaded true
     Result *result = [Result new];
-    Measurement *measurement = [self addMeasurement:result];
-    XCTAssert(![result isEveryMeasurementUploaded]);
+    [self addMeasurement:result];
+    XCTAssert([result isEveryMeasurementUploaded]);
 }
 
 - (void)testUploaded {
     //Measurement uploaded but report_id null, isEveryMeasurementUploaded false
     Result *result = [Result new];
     Measurement *measurement = [self addMeasurement:result];
     measurement.is_uploaded = true;
+    measurement.is_done = true;
     [measurement save];
     XCTAssert(![result isEveryMeasurementUploaded]);
 }
```

Comment:
Can you also explain why here the test needed to be changed?

Reply:
because now we don't care about tests who are not done. 
This is how the test should have been written since the beginning

Reply:
ack